packages/@okta/vuepress-site/docs/concepts/okta-data-model/index.md

```diff
@@ -14,20 +14,20 @@ Entities inside Okta are referred to as resources. Each Okta resource contains:
 
 * A Profile that allows you to store information about the object. For example, a user profile has an `email` attribute that contains the value for the email address.
 
-When you sign up for Okta, a private data space is created for you that is represented as an **organization** in Okta. As an Okta customer, you own your organization and everything in it: applications, directories, users, groups, policies, and so on.
+When you sign up for Okta, a private data space is created for you that is represented as an **organization** (org) in Okta. As an Okta customer, you own your org and everything in it: apps, directories, users, groups, policies, and so on.
 
-A top-level directory resource is available for your user base in your organization. You can host directories either inside Okta or replicate outside user directories, as in the case of Active Directory or LDAP.
+A top-level directory resource is available for your user base in your org. You can host directories either inside Okta or replicate outside user directories, as in the case of Active Directory or LDAP.
 
-Your end users are modeled inside Okta as [**Users**](/docs/reference/api/users/). Users are associated with an email address and a username. The username must be unique, but it can be changed (and often is when, for example, someone changes their name). The email address doesn't need to be unique and often isn't. Uniqueness for a user is determined by their Okta user ID.
+Your end users are modeled inside Okta as [**Users**](/docs/reference/api/users/). Users are associated with an email address and a username. The username must be unique, but it can be changed (and often is when, for example, someone changes their name). The email address doesn't need to be unique and often isn't. The Okta user ID determines the uniqueness for a user.
 
-A [**Group**](/docs/reference/api/groups/) is made up of users found within a directory. It can be thought of as a label applied to a set of users. A user can be a member of multiple groups. Groups are useful for representing roles, relationships, and can even be used for subscription tiers.
+A [**Group**](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Group/) is made up of users found within a directory. It can be thought of as a label applied to a set of users. A user can be a member of multiple groups. Groups are useful for representing roles, relationships, and can even be used for subscription tiers.
```

Comment:
```suggestion
A [**Group**](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Group/) is made up of users within a directory. It can be thought of as a label that's applied to a set of users. A user can be a member of multiple groups. Groups are useful for representing roles, relationships, and can even be used for subscription tiers.
```